transformers4rec/torch/model/base.py

```diff
@@ -738,7 +738,9 @@ def input_schema(self):
 
             dtype = {0: np.float32, 2: np.int64, 3: np.float32}[column.type]
             tags = column.tags
-            is_list = column.value_count.max > 0
+            dims = None
+            if column.value_count.max > 0:
+                dims = (None, column.value_count.max)
```

Comment:
Do the `value_counts` always have the same size here? If so, it should still be safe to record the second dimension as `(column.value_count.min, column.value_count.max)` just in case they differ at some point (like after we add ragged input support to T4R.)